apps/volunteers/models.py

```diff
@@ -46,7 +47,10 @@ def save(self, *args, **kwargs):
         self.active = (self.active is True)
         super(Volunteer, self).save(*args, **kwargs)
 
-
+    def get_absolute_url(self):
+        """Returns the url to access a particular volunteer instance."""
+        return reverse('volunteers:profile', args=[str(self.id)])    
+    
```

Comment:
`get_absolute_url` function for accessing the particular volunteer instance.

Reply:
This is cool.